Resources/Catalog/FileCatalogClient.py

```diff
@@ -8,36 +8,63 @@
 from types import ListType, DictType
```

Comment:
Can you profit of these deep changes to change the names of the modules related to the DIRAC file catalog... This is really confusing to have in the same directory modules that are called FileCatalog (generic abstraction), FileCatalogClientBase (base class for all catalogs) and FileCatalogClient (specific implementation)... DFCClient.py ? or DiracFileCatalogClient.py